metadata/Cadastre/plss_point_gcdb.md

```diff
@@ -0,0 +1,70 @@
+# Title
+
+Utah PLSS Point GCDB
+
+## ID
+
+6ba4ea4e-ec1e-435f-ac9e-52c638927319
+
+## Brief Summary
+
+Points dataset of the corners of the Public Land Survey System sections.
+
+## Summary
+
+Point dataset that depicts the Public Land Survey System (PLSS) for the State of Utah and is based on the Geographic Coordinate Data Base (GCDB). This dataset was created as part of the effort to provide continuous cadastre data for the state of Utah.
+
+## Description
+
+### What is the dataset?
+
+Originally established in 1785 and currently administered by the Bureau of Land Management (BLM), the PLSS is a nationwide framework for surveying, dividing, and quantifying land. This dataset represents the corner points of the PLSS and contains summary information about the coordinate location and steward who collected the data.
+
+### What is the purpose of the dataset?
+
+The PLSS corner points serve as the backbone for all descriptions of private property and public land boundaries in Utah. Maintaining the accuracy of these corners and making their locations easily accessible reduces the potential for boundary discrepancies and disputes.
+
+To facilitate this crucial task, UGRC developed the [PLSS web application](https://plss.utah.gov/) in 2013. Initially designed to provide access to PLSS monument record sheets, the app has since evolved to enable surveyors to upload their own monument record sheets. UGRC makes continual improvements to ensure the app remains a comprehensive and user-friendly tool for accessing critical survey data. You can read more about the application in our [application portfolio highlights](https://gis.utah.gov/solutions/for-government/application-portfolio-highlights/#plss-monument-records).
+
+### What does the dataset represent?
+
+Each point in this dataset represents the corner location of a PLSS section. More information about how corner data was surveyed and collected can be found on the [PLSS web application](https://plss.utah.gov/).
```

Comment:
Each point in this dataset represents a significant corner location of the PLSS,

Reply:
That's been changed!